neps/nep-0330.md

```diff
@@ -51,7 +62,21 @@ As an example, say there was an NFT contract deployed on-chain which was current
 ```ts
 type ContractSourceMetadata = {
   version: "39f2d2646f2f60e18ab53337501370dc02a5661c"
-  link: "https://github.com/near-examples/nft-tutorial"
+  link: "https://github.com/near-examples/nft-tutorial",
+  standards: [
+    {
+        standard: "nep330", 
+        version: "1.1.0"
+    },
```

Comment:
I have added reference to NEP-330 into all the examples to ensure that readers follow the advice.